src/core_plugins/metrics/public/visualizations/components/flot_chart.js

```diff
@@ -172,15 +174,17 @@ class FlotChart extends Component {
     return _.assign(opts, props.options);
   }
 
-  handleResize() {
-    const resize = findDOMNode(this.resize);
+  handleResize(width, height) {
+    this.size = { width, height };
+    console.log(this.size);
     if (!this.rendered) {
       this.renderChart();
       return;
     }
 
-    if (resize && resize.clientHeight > 0 && resize.clientHeight > 0) {
+    if (this.size.height > 0 && this.size.width > 0) {
       if (!this.plot) return;
+      console.log('here');
```

Comment:
@mattapperson Can you create a PR that removes all the `console.log`s introduced here?

Reply:
Will do